scanner/src/main/kotlin/Main.kt

```diff
@@ -183,8 +188,9 @@ object Main {
             exitProcess(1)
         }
 
-        // Make the parameter globally available.
+        // Make the parameters globally available.
         com.here.ort.utils.printStackTrace = stacktrace
+        com.here.ort.scanner.scanners.askalonoExecutablePath = askalanoPath
```

Comment:
The `printStackTrace` is assigned this way because it is originally defined in another module, and its value shall be accessed by code from the downloader. For the `askalonoPath` it would be fine to just make the var public, as it is only used inside the scanner.